README.md

```diff
@@ -88,3 +88,6 @@
 
 - [Forum](https://discourse.kedro.community/)
 - [Stackoverflow (tag:kedro)](https://stackoverflow.com/questions/tagged/kedro)
+- [kedro-plugin](https://github.com/topics/kedro-plugin) topic on GitHub
+- [#kedro](https://dev.to/t/kedro) tag on dev.to
+- [kedro-community](https://github.com/quantumblacklabs/kedro-community) github repo
```

Comment:
This changed to https://github.com/kedro-org/kedro-community/ - but we're evaluating what to do with it https://github.com/kedro-org/kedro-devrel/issues/40 do you folks still maintain this repository? Happy to contribute more stuff to it 😊 cc @WaylonWalker @fkromer

Reply:
Friendly ping @WaylonWalker @fkromer ❤️ If you don't have time to keep maintaining this, we can fork over `kedro-org` and give this some love.